java/src/com/github/bazel_contrib/contrib_rules_jvm/junit5/BazelJUnitOutputListener.java

```diff
@@ -161,7 +164,7 @@ private List<TestData> findTestCases_locked() {
               // are identified by the fact that they have no child test cases in the
               // test plan, or they are marked as tests.
               TestIdentifier id = result.getId();
-              return id.isTest() || testPlan.getChildren(id).isEmpty();
+              return id.getSource() != null || id.isTest() || testPlan.getChildren(id).isEmpty();
```

Comment:
I'm not 100% on what the right filter here. 
I am choosing source but not clear why `getChildren` is a filter
For the `@AfterAll` methods it looks like they have children.

Here is an example of children from the PR description `@AfterAll` method:
```
[TestIdentifier [uniqueId = [engine:junit-jupiter]/[class:com.library.AlwaysFailTest]/[method:doNothingTest()], parentId = [engine:junit-jupiter]/[class:com.library.AlwaysFailTest], displayName = 'doNothingTest()', legacyReportingName = 'doNothingTest()', source = MethodSource [className = 'com.library.AlwaysFailTest', methodName = 'doNothingTest', methodParameterTypes = ''], tags = [], type = TEST], TestIdentifier [uniqueId = [engine:junit-jupiter]/[class:com.library.AlwaysFailTest]/[method:executeFirst()], parentId = [engine:junit-jupiter]/[class:com.library.AlwaysFailTest], displayName = 'executeFirst()', legacyReportingName = 'executeFirst()', source = MethodSource [className = 'com.library.AlwaysFailTest', methodName = 'executeFirst', methodParameterTypes = ''], tags = [], type = TEST]]
```